minutes/2020-06-04.md

```diff
@@ -0,0 +1,93 @@
+# Meeting Minutes for June 4, 2020
+
+## Follow-up on previous meetings
+
+### ort-geovernance repository
+
+Thomas: The repository was made public using CC-BY-4.0 as license and is now
+also used for the meeting minutes.
```

Comment:
Merge this to below's "ort-governance is now public" point?

Reply:
Done.